### ISSUE TITLE
Where do I find "EPKL.exe"?

### ISSUE THREAD

**Issue by slurpey** [DreymaR/BigBagKbdTrixPKL#6]
Hi,
In your instructions, you state: Also make sure you have the EPKL.exe binary.  I might just be very newbie but where do I find this?  I've searched forums and web and I can't find this anywhere.  Thanks

**Comment by slurpey** [comment]
Ok, found it...  You need to "run Compile_EPKL.bat"...

**Comment by DreymaR** [comment]
Grats on finding the "secret mode" on your own! 🥇 Thing is, in the latest release I decided to take the executable binary (a fancy term for EPKL.exe) out of version control because that's what the cool cats do it seems. Instead, I've added it to release v1.1.2 as a downloadable asset. But as you found out, it isn't hard to compile yourself either.

In the next release, I'll update the README so people can find their .exe even without being as clever as you had to be. 😸
